litmus-portal/frontend/public/index.css

```diff
@@ -1,3 +1,4 @@
 body > #root > div {
   height: 100vh;
+  background-color: white;
```

Comment:
Is this background color change in /public necessary?

Reply:
It's necessary here because by default it will pick the `white` as background but in another project we need some different background color, but anyway it's not coming from the theme so we can remove it